erpnext/accounts/doctype/gl_entry/gl_entry.json

```diff
@@ -452,6 +452,36 @@
    "set_only_once": 0, 
    "unique": 0
   }, 
+  {
+   "allow_bulk_edit": 0, 
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "columns": 0, 
+   "fieldname": "bill_no", 
```

Comment:
Don't add extra column for bill_no in GL Entry, get the data on runtime.

Reply:
Fixed!